.gitignore

```diff
@@ -26,4 +26,6 @@ docs/
 examples_old/
 
 # jetBrains IDE ignores
-.idea
\ No newline at end of file
+.idea
+
+/bin
```

Comment:
Would you mind moving the jeyBrains stuff above "project ignores" and adding `bin/` to project ignores?